src/main/java/uk/gov/ons/ctp/response/sample/message/impl/SFTPSampleReceiver.java

```diff
@@ -1,82 +0,0 @@
-package uk.gov.ons.ctp.response.sample.message.impl;
-
-import lombok.extern.slf4j.Slf4j;
-import net.sourceforge.cobertura.CoverageIgnore;
-import org.springframework.integration.annotation.MessageEndpoint;
-import org.springframework.integration.annotation.ServiceActivator;
-import org.springframework.messaging.Message;
-import org.springframework.messaging.MessagingException;
-import org.springframework.messaging.support.GenericMessage;
-import org.springframework.messaging.support.MessageBuilder;
-
-/** Created by Kieran Wardle on 13/04/2017. */
-@Slf4j
-@MessageEndpoint
-public class SFTPSampleReceiver {
-
-  /**
-   * Confirms file rename successful for XML input file
-   *
-   * @param message success message
-   */
-  @CoverageIgnore
-  @ServiceActivator(inputChannel = "renameSuccessProcess")
-  public void sftpSuccessProcess(GenericMessage<GenericMessage<byte[]>> message) {
-    String filename = (String) message.getPayload().getHeaders().get("file_name");
-    log.debug("Renaming successful for " + filename);
-  }
-
-  /**
-   * Confirms file rename unsuccessful for XML input file
-   *
-   * @param message failure message
-   */
-  @CoverageIgnore
-  @ServiceActivator(inputChannel = "renameFailedProcess")
-  public void sftpFailedProcess(GenericMessage<MessagingException> message) {
-    String filename =
-        (String) message.getPayload().getFailedMessage().getHeaders().get("file_name");
-    log.debug("Renaming failed for" + filename);
-  }
-
-  /**
-   * Creates error file containing the reason for XML validation failure
-   *
-   * @param errorMessage failure message containing reason for failure
-   * @return Message<String> message containing cut down error message and new file names
-   */
-  @ServiceActivator(inputChannel = "pollerErrorChannel", outputChannel = "errorUploadChannel")
```

Comment:
Assuming this outputChannel does nothing?

Reply:
well, neither the input nor the output channel are declared in any of the springintegration XMLs, so I don't think those rabbit queues/topics even exist

Reply:
we know that the sample service receives its files via MIME multipart upload. Perhaps there was - in prehistoric times - a plan to use SFTP as an alternative delivery method??! Call the software historians  ;)